### PR TITLE
Add separate ha jobs for cloud 6 and 7 with  linuxbridge vxlan.

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

**Comment by aplanas** [comment]
LGTM +1

**Comment by dirkmueller** [comment]
the way I understood the card is that the request was not about compute-ha but just a normal ha cluster with vxlan

**Comment by gosipyan** [comment]
@dirkmueller thanks , i will change and update PR.

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-ha-linuxbridge-template.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=4
+            hacloud=1
+            mkcloudtarget=all_noreboot
+            networkingplugin=linuxbridge
+            networkingmode=vxlan
+            want_node_aliases=controller=2,compute=2
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-ha-linuxbridge-template.yaml]:
```bash
    clusterconfig=services='data+network+services=2'
```

missing?

Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-ha-linuxbridge-template.yaml]: also, I guess you want/need to disable ceph

**Pull request closed without merging**